### ISSUE TITLE
Unable to generate WAD File - League Patch 13.1

### ISSUE THREAD

**Issue by MrJayPockets** [Rerago/LoLWideScreenFix#7]
`Command Ran:`
```
.\LoLWideScreenFix.exe -l "D:\Riot Games\League of Legends" -o "D:\Riot Games\ultrawide-fix" -t 1441 -m 2
```

`Output:`
```
LoLWideScreenFix v0.4
=====================

[7:14:39 PM] LoL directory:     D:\Riot Games\League of Legends
[7:14:39 PM] Output directory:  D:\Riot Games\ultrawide-fix
[7:14:39 PM] target resolution :        1441p

                                                                                                                                                                                                                                                [7:14:45 PM] Create LoLCustomSkinMod:   DONE
```

The directory that it outputs the WAD file, "D:\Riot Games\cslol-manager\LoLWideScreenFix\WideScreenFix (1441p) - 1.0 (by Rerago)\WAD", is now empty. It Likely has to do with the latest 13.1 patch updates.

**Comment by Rerago** [comment]
Yes I am aware of this unfortunately. Riot has changed the structure within the wad/bin files. Therefore my current approach does not fit anymore.

But I will look into it and hopefully find a solution soon.

**Comment by lilosenpai** [comment]
> Yes I am aware of this unfortunately. Riot has changed the structure within the wad/bin files. Therefore my current approach does not fit anymore.
> 
> But I will look into it and hopefully find a solution soon.

Any news? Is it gonna be possible to fix it?

**Comment by Rerago** [comment]
Should be fixed now with [v0.5](https://github.com/Rerago/LoLWideScreenFix/releases/tag/v0.5). It may not be perfect yet. But in my test it worked well like [v0.4](https://github.com/Rerago/LoLWideScreenFix/releases/tag/v0.4).

**Comment by MrJayPockets** [comment]
Works perfectly now! Thanks so much for doing this!
